lib/App/Netdisco/Worker/Plugin/Discover/PortAccessEntity.pm

```diff
@@ -0,0 +1,18 @@
+package App::Netdisco::Worker::Plugin::Discover::PortAccessEntity;
+
+use Dancer ':syntax';
+use App::Netdisco::Worker::Plugin;
+use aliased 'App::Netdisco::Worker::Status';
+use Dancer::Plugin::DBIC 'schema';
+use App::Netdisco::Util::Worker;
+use App::Netdisco::Util::PortAccessEntity qw/update_pae_attributes/;
+
+register_worker({ phase => 'store', driver => 'snmp' }, sub {
```

Comment:
any reason this is phase 'store' and not phase 'main'?

Reply:
ah I think I've worked it out ... you want PortAccessEntry to happen after PortProperties (which deletes rows). Try naming it `App::Netdisco::Worker::Plugin::Discover::PortProperties::PortAccessEntity`

Reply:
Yes exactly, also insert. I'll try this sub-package variant.

Reply:
Worked like a charm. I also did it for the Macsuck plugin for symmetry reasons